Shared/Samples/Query feature table/QueryFeatureTableView.swift

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct QueryFeatureTableView: View {
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    /// The text in the search bar.
+    @State private var searchBarText = ""
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: model.map)
+                .alert(isPresented: $isShowingAlert, presentingError: error)
+                // Makes the search bar.
+                .searchable(text: $searchBarText, prompt: Text("Search state names"))
+                .onSubmit(of: .search, {
+                    model.currentQuery = searchBarText
+                })
```

Comment:
I was a bit surprised that the [trailing closure linter rule](https://realm.github.io/SwiftLint/trailing_closure.html) isn't triggered on this line. So it's up to you if make this change.

```suggestion
                .onSubmit(of: .search) {
                    model.currentQuery = searchBarText
                }
```